### PR TITLE
Dungeon rewards

### PR DESCRIPTION
**Pull request by AznCelebration**

### CODE REVIEW

File: src/game/phase/dungeon.h [hunk 1]
```diff
@@ -22,6 +22,15 @@ class DungeonPhase : public TimedPhase {
...
+    std::map<int, int> goal_reached_flags;
+
+    // Counter for finish order
+    int next_place;
+
+    // Vector for points, the player's ranking is 
+    // the index used for accessing the reward
+    // (0th place indicates you didn't finish)
+    std::vector<int> rewards =
+    { 0, 100, 75, 50, 25 };
```
Comment (metakirby5) [inline src/game/phase/dungeon.h:35]: Why is this 1-indexed? Would be more clear to 0-index.

**Review by metakirby5 — CHANGES_REQUESTED** [review]
Seems good besides the 1-indexing thing.

File: src/game/phase/dungeon.h (hunk 1 above)
Comment (AznCelebration) [reply src/game/phase/dungeon.h:35]: 0 is for people that didn't finish i.e. they don't get anything due to being 0th place. Then index 1 is for 1st place, onwards.

Comment (metakirby5) [reply src/game/phase/dungeon.h:35]: Oh okay, maybe I should read next time 👶

**Review by metakirby5 — APPROVED** [review]
🇿🇼

File: src/game/phase/dungeon.h (hunk 1 above)
Comment (AznCelebration) [reply src/game/phase/dungeon.h:35]: 💯

**Pull request merged**